ddtrace/_trace/context.py

```diff
@@ -234,6 +217,18 @@ def set_baggage_item(self, key: str, value: Any) -> None:
         """
         self._baggage[key] = value
 
+    def copy(self, trace_id: int, span_id: int) -> "Context":
+        """Return a shallow copy of the context with the given correlation IDs."""
+        return self.__class__(
+            trace_id=trace_id,
+            span_id=span_id,
+            meta=self._meta,
+            metrics=self._metrics,
+            lock=self._lock,
+            baggage=self._baggage,
```

Comment:
This is a pre-existing bug but copy is missing the _span_links property. Not sure why that's the case. Can we leave a comment here. I can fix it in a future PR.